woocommerce/woocommerce
# Monorepo: Folder structure

### Changes proposed in this Pull Request:

This PR moves WooCommerce plugin folders and files to `plugins/woocommerce` to reflect the new monorepo structure. The base branch is `monorepo` which we can use to incrementally add back functionality. Files and folders were moved using `git mv` to preserve history. No code changes were made and as such the plugin will be completely broken. 

### How to test the changes in this Pull Request:

Pull down this branch and ensure the relevant files were moved and files affecting the repo as a whole, such as `.gitignore` were left in place. See https://github.com/woocommerce/woocommerce-monorepo for a guide.
